akka-actor-tests/src/test/scala/akka/pattern/BackoffOnRestartSupervisorSpec.scala

```diff
@@ -91,16 +91,19 @@ class BackoffOnRestartSupervisorSpec extends AkkaSpec with ImplicitSender {
         // Exponential back off restart test
         probe.within(1.4 seconds, 2 seconds) {
           supervisor ! "THROW"
-          // numRestart = 0 ~ 200 millis
-          probe.expectMsg(300 millis, "STARTED")
+          // numRestart = 0: expected delay ~200 millis
+          probe.expectNoMessage(200 millis)
+          probe.expectMsg(150 millis, "STARTED")
```

Comment:
would it be wrong to have this longer? feels tight on the poor moxies

is the above `within(1.4 seconds, 2 seconds)` still useful, or remove?